bazel/repository_locations.bzl

```diff
@@ -1448,6 +1448,21 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         license = "MIT",
         license_url = "https://github.com/protocolbuffers/utf8_range/blob/{version}/LICENSE",
     ),
+    com_github_maxmind_libmaxminddb = dict(
+        project_name = "maxmind_libmaxminddb",
```

Comment:
Does this comply with https://github.com/envoyproxy/envoy/blob/main/DEPENDENCY_POLICY.md#new-external-dependencies?

Reply:
Good point, I need to file the dedicated issue for dependency review. Working on it

Reply:
Waiting for input from libmaxminddb maintainers: https://github.com/maxmind/libmaxminddb/issues/319

Reply:
I think I'm reasonably convinced this is a dependency we can on-board. @moderation @phlax any objection?

Reply:
ping @phlax @moderation for opinions

Reply:
@phlax @moderation ping.